### PR TITLE
Test for not featured contacts

### PR DESCRIPTION
**Pull request by rajputanuj31**
Add a test for not featured contacts.

### CODE REVIEW

File: tests/System/integration/site/components/com_contact/Featured.cy.js [hunk 1]
```diff
@@ -15,4 +15,15 @@ describe('Test that the front page', () => {
         cy.contains('automated test contact 4');
       });
   });
+  it('can not display not featured contacts', () => {
```
Comment (laoneo) [inline tests/System/integration/site/components/com_contact/Featured.cy.js:19]: Can you add here a blank line?

**Comment by laoneo** [comment]
There are also some cs errors?

File: tests/System/integration/site/components/com_contact/Featured.cy.js (hunk 1 above)
Comment (rajputanuj31) [reply tests/System/integration/site/components/com_contact/Featured.cy.js:19]: Done.

**Comment by laoneo** [comment]
Thanks!

**Comment by rajputanuj31** [comment]
Thanks for merging it.

**Pull request merged**